ImageConverter/ImageConverter/Ppm/PpmImageReader.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Text;
+
+namespace ImageConverter.Ppm;
+
+public class PpmImageReader : IImageReader
+{
+    public Image Read(string source)
+    {
+        Pixel[,] pixelmap;
+        using (StreamReader streamReader = new StreamReader(source))
+        {
+            string fileFormatNumber = ReadUntilDelimiter(streamReader);
+            if (fileFormatNumber != "P3")
+                throw new ArgumentException("This file is not plain PPM format");
+            int width = int.Parse(ReadUntilDelimiter(streamReader));
+            int height = int.Parse(ReadUntilDelimiter(streamReader));
+            int colorDepth = int.Parse(ReadUntilDelimiter(streamReader));
+            pixelmap = new Pixel[height, width];
+            for (int i = 0; i < height; i++)
+            {
+                for (int j = 0; j < width; j++)
+                {
+                    string red = ReadUntilDelimiter(streamReader);
+                    byte redValue = byte.Parse(red);
```

Comment:
It will throw an exception if the file contains letters

Reply:
right, cover this method with try-catch in 790fca6